distributed/scheduler.py

```diff
@@ -1775,6 +1775,19 @@ def _clear_task_state(self) -> None:
         ):
             collection.clear()  # type: ignore
 
+    @property
+    def fully_idle(self) -> bool:
```

Comment:
nit: Is there a way we can stress the semantic difference between `idle` and `fully_idle` (e.g., by renaming `fully_idle -> is_idle` or `idle -> idle_workers`)?

Reply:
renamed `fully_idle -> is_idle`